.eslintrc

```diff
@@ -235,6 +236,14 @@
         "before": false
       }
     ],
+    "import/order": [
```

Comment:
Add `alphabetize`(descending order, not sure about the case) and `newlines-between` (probably `always`) (https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md).